FlowCrypt/Functionality/Services/Local Private Key Services/InMemoryPassPhraseStorage.swift

```diff
@@ -15,13 +15,16 @@ final class InMemoryPassPhraseStorage: PassPhraseStorageType {
     let timeoutInSeconds: Int
     let calendar = Calendar.current
     let passPhraseProvider: InMemoryPassPhraseProviderType
+    let encryptedStorage: EncryptedStorageType
 
     init(
         passPhraseProvider: InMemoryPassPhraseProviderType = InMemoryPassPhraseProvider.shared,
+        encryptedStorage: EncryptedStorageType,
         timeoutInSeconds: Int = 4*60*60 // 4 hours
     ) {
         self.passPhraseProvider = passPhraseProvider
         self.timeoutInSeconds = timeoutInSeconds
+        self.encryptedStorage = encryptedStorage
     }
 
     func save(passPhrase: PassPhrase) {
```

Comment:
either `PassPhrase` should contain `email` which should be used, or this method needs an `email` input and separate them.

As I said - all of these methods need an email input and need to keep pass phrases separate.

The `update`, `remove` methods too.